Apps/Admin/Tests/Functional/cypress/support/commands.js

```diff
@@ -10,6 +10,7 @@
 require("cy-verify-downloads").addCustomCommand();
 
 const openIdConnectClientId = "hg-admin-blazor";
+const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/;
```

Comment:
I think the comment on the Stack Overflow answer is correct and this regular expression doesn't exactly work as intended.

A simpler solution is to replace the regex check with `err.message.includes("ResizeObserver loop limit exceeded")`.